tests/ga/test_update.py

```diff
@@ -2195,28 +2191,28 @@ def test_upload_vm_status_even_on_failed_goal_state_fetch(self):
             # returning None forces the default value. We return twice, the first for 
             # protocol.detect during creation of the mock wire protocol, the second for
             # the initial call in run().
-            ReportStatusTestCase._goal_state_handler.return_vals = [ None, None ]
+            ReportStatusTestCase._fail_goal_state_fetch.return_vals = [ None, None ]
 
-            with ReportStatusTestCase._mock_update_handler(http_get_handler=ReportStatusTestCase._goal_state_handler) as update_handler:
+            with ReportStatusTestCase._mock_update_handler(http_get_handler=ReportStatusTestCase._fail_goal_state_fetch) as update_handler:
                 update_handler.run(debug=True)
 
                 mock_protocol = update_handler.protocol_util.get_protocol()
                 self.assertEqual(mock_protocol.mock_wire_data.call_counts['/StatusBlob'], 1,
                     "Expected a single status blob to be uploaded")
         finally:
             # clean up the static variable
-            del ReportStatusTestCase._goal_state_handler.return_vals
+            del ReportStatusTestCase._fail_goal_state_fetch.return_vals
     
-    def test_update_status_for_cached_goal_state_on_failed_fetch(self):
+    def test_upload_status_for_cached_goal_state_on_failed_fetch(self):
```

Comment:
I missed this earlier, but we generally follow this naming convention for our tests - `test_it_should_*`. Please rename this to the same pattern